src/esg_fastapi/api/models.py

```diff
@@ -61,236 +63,1132 @@ class ESGSearchQueryBase(BaseModel):
+        ),
+        openapi_examples={
+            "HTTPServer": {"summary": "HTTPServer", "description": "Datasets available via direct HTTP download.", "value": "HTTPServer"},
+            "Globus": {"summary": "Globus", "description": "Datasets available via Globus file transfer.", "value": "Globus"},
+            "OPENDAP": {"summary": "OPeNDAP", "description": "Datasets available via OPeNDAP.", "value": "OPENDAP"},
+            "GridFTP": {"summary": "GridFTP", "description": "Datasets available via GridFTP.", "value": "GridFTP"},
+        },
+    )
+    activity: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Activity (CMIP5)",
+        description=(
+            "Identifies the model intercomparison `activity`. For CMIP5, this was the primary identifier for the project (e.g., `CMIP5`, "
+            "`TAMIP`, `CFMIP`, `PMIP`) <sup>1</sup>. In CMIP6, this role is largely taken over by `activity_id` and `mip_era`.<br><br>"
+            "<small><sup>1</sup>: [CMIP5 Data Reference Syntax (DRS)]"
+            "(https://pcmdi.llnl.gov/mips/cmip5/docs/cmip5_data_reference_syntax.pdf), Section 2.3</small>"
+        ),
+        openapi_examples={
+            "CMIP5": {"summary": "CMIP5", "description": "Data from CMIP5 activity.", "value": "CMIP5"},
+            "TAMIP": {"summary": "TAMIP", "description": "Data from TAMIP activity.", "value": "TAMIP"},
+            "CFMIP": {"summary": "CFMIP", "description": "Data from CFMIP activity.", "value": "CFMIP"},
+            "PMIP": {"summary": "PMIP", "description": "Data from PMIP activity.", "value": "PMIP"},
+        },
+    )
+    activity_drs: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Activity DRS (CMIP6)",
+        description="`Activity` identifier as used in the CMIP6 Data Reference Syntax (DRS) for constructing file paths and IDs. <sup>1</sup>"
+        "<br><br><small><sup>1</sup>: [CMIP6 Global Attributes, DRS, Filenames, Directory Structure, and CV's]"
+        "(https://wcrp-cmip.github.io/WGCM_Infrastructure_Panel/Papers/CMIP6_global_attributes_filenames_CVs_v6.2.7.pdf)</small>",
+        openapi_examples={
+            "DCPP": {"summary": "DCPP", "description": "Activity identifier for DCPP.", "value": "DCPP"},
+            "PAMIP": {"summary": "PAMIP", "description": "Activity identifier for PAMIP.", "value": "PAMIP"},
+            "ScenarioMIP": {"summary": "ScenarioMIP", "description": "Activity identifier for ScenarioMIP.", "value": "ScenarioMIP"},
+        },
+    )
+    activity_id: OptionalParam[MultiValued[str]] = Query(
+        None,
+        title="Activity ID (CMIP6, Obs4MIPs)",
+        description=(
+            "Identifies the specific `activity` or MIP associated with the dataset (e.g., `CMIP`, `PMIP`, `DCPP`, `obs4MIPs`). It "
+            "replaces the broader `activity` field from CMIP5 and can be a space-separated list in CMIP6 <sup>1, 2</sup>.<br><br>"
```

Comment:
Are we sure it can be a space-separated list in CMIP6? That doesn't sound right to be, each dataset should have only a single activity_id